src/components/visualizations/chart/chartOptionsBuilder.ts

```diff
@@ -891,7 +901,7 @@ export function findAttributeInDimension(
         });
 }
 
-export function getDrillContext(stackByItem: any, viewByItem: any, measures: AFM.IMeasure[], afm: AFM.IAfm) {
+export function getDrillContext(stackByItem: any, viewByItem: any, measures: any[], afm: AFM.IAfm) {
```

Comment:
why `measures: AFM.IMeasure[]` is now `any`?

Reply:
ignore that, it see the (typing) mess around

Reply:
Better to put `any` than wrong type, that was used before and due to obfuscated TS was invisible to TSC... Needs refactor. Wont fix in this PR.